source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -467,6 +467,27 @@
                 };
             });
         });
+
+        this.showCalificationsComments = ko.computed(function () {
+            var groupsItemsShowComments = _.map(self.groups, function (group) {
+                if (_.every(group.items, function (item) { return item.showComments(); })) {
+                    return true;
+                }
+                else if (_.some(group.items, function (item) { return item.showComments(); })) {
+                    return null;
+                }
+                else {
+                    return false;
+                }
+            });
+            if (_.every(groupsItemsShowComments, function (itemsShowComments) { return itemsShowComments; })) {
+                return true;
+            }
+            else if (_.every(groupsItemsShowComments, function (itemsShowComments) { return itemsShowComments === false; })) {
+                return false;
+            }
+            else return null;
```

Comment:
It is more complex that I imagine, maybe if you flat the items before evaluate it could look simpler. Right now, at a first glance I cannot understand what is happening here.